instances/instance_log.go

```diff
@@ -43,3 +45,189 @@ func (c *client) Log(ctx context.Context, id string, offset int, limit int) (*kc
 
 	return resp, nil
 }
+
+// TailLogs implements InstancesService.
+func (c *client) TailLogs(ctx context.Context, id string, follow bool, tail int) (chan string, chan error, error) {
+	ctx, cancel := context.WithCancel(ctx)
+
+	var (
+		logChan     = make(chan string)
+		errChan     = make(chan error)
+		startOffset int
+	)
+
+	// Start a goroutine to concurrently fetch logs and send them over the logs
+	// channel.
+	go func() {
+		// If tail is set greater than zero, start by fetching the logs in reverse
+		// order, separating by newline and appending to a buffer. Once the buffer
+		// size reaches the length of tail, print the lines by reversing the buffer
+		// again.
+		if tail > 0 {
+			buf := make([]string, 0)
```

Comment:
I think you could use `bytes.Buffer`, not sure how faster/better it would make things

Reply:
Nvm, not this, it's not enough parsing anyway